src/Wrap.js

```diff
@@ -11,6 +11,7 @@ export type WrapProps = {
 const Wrap = (props: WrapProps): React.Element<*> => {
   const idClip = `${props.uniquekey}-idClip` || uid()
   const idGradient = `${props.uniquekey}-idGradient` || uid()
+  const showAnimation = props.animation ? true : false
```

Comment:
You don't need this because `props.animation` is already a boolean

Reply:
Point taken 👍